src/NUnitFramework/framework/Internal/Execution/SimpleWorkItem.cs

```diff
@@ -140,6 +140,13 @@ private TestCommand MakeTestCommand()
                 foreach (var attr in method.GetCustomAttributes<IApplyToContext>(true))
                     command = new ApplyChangesToContextCommand(command, attr);
 
+                // Add construct and optionally dispose command in case of instance per test case.
```

Comment:
```suggestion
                // Add a construct command and optionally a dispose command in case of instance per test case.
```

Helps me understand better.